### PR TITLE
adapt to support epsilla cloud

### PR DESCRIPTION
**Pull request by eric-epsilla**
Description

- support epsilla cloud in langchain
- update related [documents](https://python.langchain.com/v0.2/docs/integrations/vectorstores/epsilla/)

### CODE REVIEW

**Review by baskaryan — APPROVED** [review]

**Comment by eric-epsilla** [comment]
@baskaryan please review, thanks

**Comment by eric-epsilla** [comment]
@baskaryan @sbusso @jarib @zeke please help review, thanks

File: docs/docs/integrations/vectorstores/epsilla.ipynb [hunk 1]
```diff
@@ -120,6 +120,62 @@
     "print(docs[0].page_content)"
    ]
   },
+  {
+   "cell_type": "code",
```
Comment (ccurme) [inline docs/docs/integrations/vectorstores/epsilla.ipynb:124]: This cell should be markdown

**Review by ccurme — APPROVED** [review]

File: docs/docs/integrations/vectorstores/epsilla.ipynb (hunk 1 above)
Comment (eric-epsilla) [reply docs/docs/integrations/vectorstores/epsilla.ipynb:124]: got it, will fix it now

Comment (eric-epsilla) [reply docs/docs/integrations/vectorstores/epsilla.ipynb]: Done @ccurme

**Comment by eric-epsilla** [comment]
the vercel deployment still failed after I haved rollbacked the file epsilla.ipynb's content to the same with main branch of langchain, please help to check what's the failed error of vercel deployment (I cannot check it), thanks!

by the way, can I find you on discord or please create a channel for us on discord? so we can quick fix this issue and merge this code. 

Many thanks @baskaryan @ccurme @efriis

**Comment by eric-epsilla** [comment]
I created another pr, https://github.com/langchain-ai/langchain/pull/26065, only 3 lines changed, but still failed at vercel deployment failed!
Would you please help check why it failed? @baskaryan @ccurme

**Comment by ccurme** [comment]
Closing in favor of https://github.com/langchain-ai/langchain/pull/26065

**Pull request closed without merging**